src/@types/global.d.ts

```diff
@@ -90,6 +90,9 @@ declare global {
         mxUIStore: UIStore;
         mxSetupEncryptionStore?: SetupEncryptionStore;
         mxRoomScrollStateStore?: RoomScrollStateStore;
+        grecaptcha: any;
+        // eslint-disable-next-line
+        mx_on_recaptcha_loaded: () => void;
```

Comment:
Blocking on https://github.com/matrix-org/matrix-react-sdk/pull/6401 which improves this area